README.md

```diff
@@ -55,23 +55,28 @@ Read more at https://www.terraform.io/docs/language/providers/requirements.html
 
 ### From source
 
-Clone repository to `$GOPATH/src/github.com/cloudamqp/terraform-provider-cloudamqp`
+Clone repository to `/home/USER/code/cloudamqp/terraform-provider-cloudamqp`
 
-Change directory and build the provider from make. This will call `go install` and install the plugin under `$GOPATH/bin`.
+Change directory and build the provider binary using `go build`.
 
-```sh
-$ cd $GOPATH/src/github.com/cloudamqp/terraform-provider-cloudamqp
-$ make build
+Setup [.terraformrc](https://developer.hashicorp.com/terraform/cli/config/config-file) so that terraform will use the binary that we just build instead of the one from the registry
+
+```
+provider_installation {
+  dev_overrides {
+    "hashicorp/cloudamqp" = "/home/USER/code/cloudamqp/terraform-provider-cloudamqp"
```

Comment:
```suggestion
    "cloudamqp/cloudamqp" = "/home/USER/code/cloudamqp/terraform-provider-cloudamqp"
```
This needs to be changed to what our provider is named. Can easily be tested with changed log output, `export TF_LOG_PROVIDER=INFO` before running `plan`. Output shows which provider plugin being used. 

As you describe below, no need to initialize the provider. The dev_overrides will kick in and pick up the provider from "/home/USER/code/cloudamqp/terraform-provider-cloudamqp".